core-bundle/contao/templates/js/js_accordion.html5

```diff
@@ -0,0 +1,57 @@
+<?php
+
+// Add the handorgel style sheet
+$GLOBALS['TL_CSS'][] = $this->asset('css/handorgel.min.css', 'contao-components/handorgel');
+
+?>
+
+<script src="<?= $this->asset('js/handorgel.min.js', 'contao-components/handorgel') ?>"></script>
+<script>
+  (function () {
+    const addNextAccordions = (el, accordion) => {
+      if (el && el.classList.contains('ce_accordion')) {
+        const toggler = el.querySelector('.toggler');
+        const element = el.querySelector('.accordion');
+
+        if (!toggler || !element) {
+          return;
+        }
+
+        const button = document.createElement('button');
```

Comment:
I guess we have to use a button for accessibility, right? Because it sure doesn‘t look nice:

<img width="713" alt="" src="https://github.com/contao/contao/assets/1192057/f7687957-b8f0-4502-9f08-c378c5dac099">